test/test_sampledata.py

```diff
@@ -85,8 +85,8 @@ def test_create_protocol(self):
         print(f'Wrote file as {temp_name}')
 
         comparison_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'testfiles', 'igem_ludox_data_test.nt')
-        # with open(comparison_file, 'w') as f:
-        #     f.write(doc.write_string(sbol3.SORTED_NTRIPLES).strip())
+        with open(comparison_file, 'w') as f:
```

Comment:
This will cause the test to always pass.  I usually uncomment in order to update the rubric.  I think it should be commented out for actual testing.